tests/file_writer/test_file_writer.py

```diff
@@ -289,46 +279,14 @@ def test_FileWriterProcess__prevent_any_uploads_with_no_stored_customer_settings
 
     file_writer_process._stored_customer_settings = None  # pylint: disable=protected-access
 
-    file_writer_process._start_new_file_uploads()  # pylint: disable=protected-access
+    file_writer_process._start_new_file_upload()  # pylint: disable=protected-access
     spied_auto_delete.assert_not_called()
     spied_thread_start.assert_not_called()
 
     file_writer_process._check_upload_statuses()  # pylint: disable=protected-access
     spied_thread_join.assert_not_called()
 
 
-@pytest.mark.parametrize(
-    "upload_status, auto_delete",
-    [("__upload failed", True), ("_upload complete", False)],
-)
-def test_FileWriterProcess__correctly_handles_when_file_upload_is_successful_and_auto_delete_is_false(
-    four_board_file_writer_process, upload_status, auto_delete, mocker
-):
-
-    file_writer_process = four_board_file_writer_process["fw_process"]
-    from_main_queue = four_board_file_writer_process["from_main_queue"]
-
-    spied_delete_files = mocker.spy(file_writer_process, "_delete_local_files")
-    mocker.patch.object(os, "listdir", return_value=["test_file"])
-    mocker.patch.object(os, "remove", autospec=True)
-    mocker.patch.object(os, "rmdir", autospec=True)
-    mocker.patch.object(shutil, "move", autospec=True)
-    mocker.patch.object(file_uploader.ErrorCatchingThread, "errors", autospec=True, return_value=False)
-    mocker.patch.object(file_uploader.ErrorCatchingThread, "is_alive", autospec=True, return_value=False)
-    mocker.patch.object(
-        file_uploader.ErrorCatchingThread, "get_upload_status", autospec=True, return_value="upload complete"
-    )
-
-    GENERIC_UPDATE_CUSTOMER_SETTINGS["config_settings"]["auto_delete_local_files"] = auto_delete
-    GENERIC_UPDATE_CUSTOMER_SETTINGS["config_settings"]["auto_upload_on_completion"] = True
-    this_command = copy.deepcopy(GENERIC_UPDATE_CUSTOMER_SETTINGS)
-    put_object_into_queue_and_raise_error_if_eventually_still_empty(this_command, from_main_queue)
-    invoke_process_run_and_check_errors(file_writer_process)
-
-    assert (len(spied_delete_files.call_args_list) > 0) is auto_delete
-    assert len(file_writer_process._upload_threads_container) == 0  # pylint: disable=protected-access
-
-
 def test_FileWriterProcess__setup_before_loop__calls_super(four_board_file_writer_process, mocker):
     # pylint: disable=protected-access
```

Comment:
Be careful not to dd pylint disables on their own line, that can have unintended behavior